pygameMenu/sound.py

```diff
@@ -69,7 +69,7 @@
 _SOUND_EXAMPLE_OPEN_MENU = __sounddir.format(__actualpath, 'open_menu')
 
 # Stores global reference that marks sounds as initialized
-SOUND_INITIALIZED = [False]
+_SOUND_INITIALIZED = [False]
```

Comment:
It could be a class attribute of the `Sound` class. Global variables are generally not intended to be private.

```python
class Sound(object):

    sound_initialized = [0]

    def __init__(self, ....
```

then access using `Sound.sound_initialized`

Reply:
> Good improvement!
> Personally I don't like repeated patterns, it makes me think to my old Windows 3.11 wallpapers, yes it make me old ;-). But probably some people will want them.

hahaha I started with w98 playing captain claw and quake

Reply:
> It could be a class attribute of the `Sound` class. Global variables are generally not intended to be private.
> 
> ```python
> class Sound(object):
> 
>     sound_initialized = [0]
> 
>     def __init__(self, ....
> ```
> 
> then access using `Sound.sound_initialized`

Thay may work on Java and python 3, but python 2 throws an exception